src/hooks/useSelector.js

```diff
@@ -100,7 +78,10 @@ export function useSelector(selector, deps) {
         latestSubscriptionCallbackError.current = err
       }
 
-      forceRender({})
+      setState(prev => ({
```

Comment:
have you actually run this in an application instead of just in a test? doesn't `setState` inside an effect that executes on each render cause an infinite render loop?

Reply:
> have you actually run this in an application instead of just in a test

Yep. Now I have.

> doesn't setState inside an effect that executes on each render cause an infinite render loop

This is not inside the render. If this caused an infinite loop, the forceRender would also cause an infinite loop.